Podcast/SeriesDetailHeaderView.swift

```diff
@@ -0,0 +1,234 @@
+//
+//  SeriesDetailHeaderView.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 2/13/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+protocol SeriesDetailHeaderViewDelegate {
+    func subscribeButtonPressed(subscribed: Bool)
```

Comment:
This needs to be name spaced so we know the delegate method is specifically for this protocol:

Should be in the format seriesDetailHeaderViewDidPressSubscribeButton(..)